hostnat/watcher.go

```diff
@@ -39,16 +41,23 @@ type watcher struct {
 // MASQRule is used to store the needed information for building
 // a masquerading rule
 type MASQRule struct {
-	Subnet string
-	Bridge string
+	Subnet          string
+	Bridge          string
+	IsPerHostSubnet bool
 }
 
 func (p MASQRule) iptables() []byte {
 	buf := &bytes.Buffer{}
 	buf.WriteString(fmt.Sprintf("-A %s -d %s -s %s -j ACCEPT\n", natChain, os.Getenv("METADATA_IP"), p.Subnet))
-	buf.WriteString(fmt.Sprintf("-A %s -p tcp -s %s ! -o %s -j MASQUERADE --to-ports 1024-65535\n", natChain, p.Subnet, p.Bridge))
-	buf.WriteString(fmt.Sprintf("-A %s -p udp -s %s ! -o %s -j MASQUERADE --to-ports 1024-65535\n", natChain, p.Subnet, p.Bridge))
-	buf.WriteString(fmt.Sprintf("-A %s -s %s ! -o %s -j MASQUERADE\n", natChain, p.Subnet, p.Bridge))
+	if p.IsPerHostSubnet {
```

Comment:
No need of introducing this. Just install the ipset by default in `plugin-manager`. This doesn't get populated for ipsec/vxlan. `per-host-subnet` will populate the ipset.